client/src/global/components/composed/EntityGroup/index.js

```diff
@@ -3,35 +3,41 @@ import PropTypes from "prop-types";
 import EntityBox from "global/components/atomic/EntityBox";
 import ThumbnailGrid from "global/components/composed/ThumbnailGrid";
 import EntityThumbnail from "global/components/atomic/EntityThumbnail";
+import { Link } from "react-router-dom";
 import * as Styled from "./styles";
 
-export default function EntityGroup({ entities, title, link = false }) {
+export default function EntityGroup({ entities, title, to }) {
   return (
-    <Styled.Wrapper>
-      <EntityBox>
-        <Styled.GroupHeader $link={link}>
-          <Styled.HeaderText>{title}</Styled.HeaderText>
-          {link && <Styled.Icon icon="ArrowLongRight16" size={24} />}
-        </Styled.GroupHeader>
-        <ThumbnailGrid minColumns={4} minItemWidth="180px">
-          {({ stack }) =>
-            entities.map(entity => (
-              <EntityThumbnail
-                entity={entity}
-                stack={stack}
-                key={entity.attributes.slug}
-              />
-            ))
-          }
-        </ThumbnailGrid>
-      </EntityBox>
-    </Styled.Wrapper>
+    <EntityBox
+      style={{
+        "--EntityBox-Container-padding-block": "30px",
+        "--EntityBox-Background-padding-block": "20px",
+        "--EntityBox-Background-padding-inline": "60px"
+      }}
+    >
```

Comment:
Using `style` should be a last resort, meaning we shouldn't build it into `EntityGroup`. The reason is that styles inlined in HTML have highest specificity, so couldn't be overridden by any other styled components (except by using `!important`). If you want to pass CSS var values to EntityBox, I think you should expose `className` as a prop on EntityBox and use it as a styled component in EntityGroup.